### PR TITLE
Support ad directory objects

### LINKED ISSUE #391: Azure Active Directory
Resource Support for Azure Active Directory
Coverage:
1.Dev
2.Test
3.Documentation

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Support ad directory objects
### Issues Resolved

#391 

### Check List

- [x] New functionality includes tests
- [x] All tests pass
- [x] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/azure_active_directory_objects.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+describe azure_active_directory_objects do
+  #...
+end
+```
+
+## Properties
+
+|Property       | Description                                                                          | Filter Criteria<superscript>*</superscript> |
+|---------------|--------------------------------------------------------------------------------------|-----------------|
+| values        | A list of the unique directory object ids.                                           | `id`            |
```
Comment (soumyo13) [inline docs/resources/azure_active_directory_objects.md:40]: id should be changed to value.

**Review by soumyo13 — COMMENTED** [review]
Please check the comment.

**Review by IanMadd — APPROVED** [review]

File: docs/resources/azure_active_directory_objects.md (hunk 1 above)
Comment (sathish-progress) [reply docs/resources/azure_active_directory_objects.md]: done

**Comment by sathish-progress** [comment]
> Please check the comment.

Please review again..

**Review by sa-progress — APPROVED** [review]

File: terraform/variables.tf [hunk 2]
```diff
@@ -145,4 +145,8 @@ variable "inspec_db_migration_service" {
 
 variable "inspec_container_group_name" {
   default = "inspec_container_trial"
+}
+
+variable "sample_directory_object" {
+  default = "adc07321-ef2b-44d5-a210-559aa5f10f2d"
```
Comment (soumyo13) [inline terraform/variables.tf:151]: Do we have to specify the hardcoded value? In AWS we specify the hardcoded values in test/integration/configuration/aws_inspec_config.rb

**Review by soumyo13 — APPROVED** [review]
LGTM.

**Pull request merged**